itest/rust/src/builtin_tests/geometry/quaternion_test.rs

```diff
@@ -28,4 +29,69 @@ fn quaternion_from_xyzw() {
     assert_eq!(quat.w, 0.8924);
 }
 
+#[itest]
+fn quaternion_from_angle_axis() {
+    let quat = Quaternion::from_angle_axis(Vector3::new(0.0, 0.0, 1.0).normalized(), 1.0);
+
+    // Taken from doing this in gdscript
```

Comment:
```suggestion
    // Taken from doing this in GDScript.
```

We try to end comments in full stops (except end-of-line ones).
Same for occurrences below.